booksApi.ts

```diff
@@ -0,0 +1,21 @@
+import { Alert } from "react-native";
+import { setBooks } from "./bookSlice";
+
+export const getBooks = async () => {
+  try {
+    const response = await fetch("https://example.com/api/books");
+    const books: Book[] = await response.json();
+    setBooks(books);
```

Comment:
Lets change to this
```suggestion
    const response: GetBooksResponse = await response.json();
    setBooks(response.books);
```

Reply:
@bbaoNg97  what should we name it then
`const response = await fetch("https://example.com/api/books");`

Reply:
booksReponse/ response also can